examples/complete/vendor/vendor2.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: atmos/v1
+kind: AtmosVendorConfig
+metadata:
+  name: atmos-vendor-config-2
```

Use `example` so it's clear it can be anything. 

```suggestion
  name: example-vendor-config-2
```